contrib/intel/jenkins/tests.py

```diff
@@ -211,6 +211,39 @@ def execute_cmd(self, shmem_testname):
         outputcmd = shlex.split(command)
         common.run_command(outputcmd)
 
+class ZeTest(Test):
+    def __init__(self, jobname, buildno, testname, core_prov, fabric,
+                 hosts, ofi_build_mode, util_prov=None):
+
+        super().__init__(jobname, buildno, testname, core_prov, fabric,
+                         hosts, ofi_build_mode, util_prov)
+        self.fabtestpath = "{}/bin".format(self.libfab_installpath)
+        self.fabtestzepath = "{}".format(ci_site_config.ze_src)
```

Comment:
I would prefer if we could put the script in the common script directory (currently called mpi_testpath) and rename the variable to script_path or something similar.